.github/workflows/test.yml

```diff
@@ -101,6 +104,60 @@ jobs:
 
           skopeo copy oci-archive:dotnet-aspnet.tar docker-daemon:${{ env.aspnet-image-name }}
 
+      - name: Generate SBOM for the .NET runtime deps container image
+        run: |
+          set -x
+          archs=`cat rockcraft.*.dotnet-deps-7.0-${{ matrix.ubuntu-release }}.yaml | shyaml get-value platforms | shyaml keys`
+          buildx_platforms="linux/$(echo ${archs} | sed 's/ /,linux\//g')"
+
+          docker buildx build \
+            --platform=${buildx_platforms} \
+            --output type=oci,dest=dotnet-deps-sbom.tar \
+            -f dotnet-deps/Dockerfile.${{ matrix.ubuntu-release }} \
+            --target sbom-prep \
+            dotnet-deps
+
+          skopeo copy oci-archive:dotnet-deps-sbom.tar docker-daemon:dotnet-deps:sbom
+          syft packages -o spdx-json docker:dotnet-deps:sbom > dotnet-deps-${{ matrix.ubuntu-release }}.sbom.json
```

Comment:
this is only gonna do it for the host's architecture. so we're missing SBOMs for non-amd64 archs

Reply:
Nice catch! Thanks. Updated in recent commits.